topic_modeling/scrape_training_data/scrape_articles.py

```diff
@@ -50,10 +50,19 @@ def test_simple_get():
 
 
 def test_log_error():
-    assert log_error('This is a test') == print('This is a test')
+    assert log_error('This is a sentence') == print('This is a sentence')
```

Comment:
not sure if that is testing anything besides that print and log_error  both return None?

Reply:
it's a very dumb test for a very dumb function

Reply:
you would be surprised how often dumb functions cause surprisingly qualitatively the same bad effect - a CRASH of the entire program or abnormal behavior.   I guess the idea here was to test output being produced. See https://docs.pytest.org/en/latest/capture.html#accessing-captured-output-from-a-test-function on how to do it pytest way